Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -171,6 +181,14 @@ def check_feed(client: Client) -> str:
 def main():
     params = demisto.params()
     server_url = (params.get('server_url')).rstrip()
+    default_headers = params.get('default_headers', USER_AGENT)
```

Comment:
`USER_AGENT` isn't the required JSON - please change it to a JSON with the header key.

Reply:
done